theme/styles/snippets/add-to-cart-container.scss

```diff
@@ -0,0 +1,29 @@
+.AddToCartContainer {
+  &__select-wrapper, &__quantity-wrapper {
+    overflow: hidden;
+    position: relative;
+    
+      &::after {
+        content: "▼";
```

Comment:
Currently using this unicode character but will confirm with XXIX if we can use this or need to use the svg triangle from Figma.